e2e/__tests__/jestChangedFiles.test.ts

```diff
@@ -21,7 +21,7 @@ const HG = 'hg --config ui.username=jest_test';
 
 const gitVersionSupportsInitialBranch = (() => {
   const {stdout} = run(`${GIT} --version`);
-  const gitVersion = stdout.split(' ').slice(-1)[0];
+  const gitVersion = stdout;
```

Comment:
This was necessary fix since on my machine, it gives `git version 2.24.3 (Apple Git-128)`